doc/plugin_agent_svidstore_sconecas_sessionmanager.md

```diff
@@ -102,7 +109,7 @@ secrets:
 **Bundle session template**
 
 ```yaml
-name: spire-ca
+name: spire-ca-<\trust-bundle-session-name>
```

Comment:
It seems the Agent fails to replace `<\trust-bundle-session-name>`, which leaves an invalid session it tries to push. I'm using the template mentioned in the documentation. As far as I can see I'm using a valid entry that definitely has a value for `sconecas_sessionmanager:trust_bundle_session_name`:

### Agent's template 

```bash
  ca_session_template.yaml: |
    name: spire-ca-<\trust-bundle-session-name>
    version: "0.3"
    predecessor: <\predecessor>
    secrets:
      - name: spire-ca
        kind: x509-ca
        export_public: true
        value: |
            <\trust-bundle>
```

### Server's entries

```bash
/opt/spire # ./bin/spire-server entry show
Found 2 entries
Entry ID         : f39f0ab1-3a36-4c44-ad9c-0a4295ea5e12
SPIFFE ID        : spiffe://example.org/client
Parent ID        : spiffe://example.org/spire/agent/k8s_sat/minikube/e2f19904-3b7e-4fcf-8336-cbd6010742ee
Revision         : 0
TTL              : default
Selector         : k8s:container-name:spiffe-aware-client

Entry ID         : 86c47b94-2a48-45fd-a683-97e89e5245fc
SPIFFE ID        : spiffe://example.org/scone-service
Parent ID        : spiffe://example.org/spire/agent/k8s_sat/minikube/e2f19904-3b7e-4fcf-8336-cbd6010742ee
Revision         : 0
TTL              : default
Selector         : sconecas_sessionmanager:session_hash:345568ee21aad083bc541acd4845d1bf1cd6e0f014c2348086e4f2bced1adf20
Selector         : sconecas_sessionmanager:session_name:benardis-service2
Selector         : sconecas_sessionmanager:trust_bundle_session_name:bundle-benardis-service2
StoreSvid        : true
```

### Agent's log (failing to replace <\trust-bundle-session-name>) 

```bash
time="2021-11-12T12:30:31Z" level=info msg="Posting Session:" external=false name: spire-ca-spire-ca-<\trust-bundle-session-name>
version: "0.3"
predecessor: ~
secrets:
  - name: spire-ca
    kind: x509-ca
    export_public: true
    value: |
        -----BEGIN CERTIFICATE-----
        MIIBzDCCAVOgAwIBAgIJAJM4DhRH0vmuMAoGCCqGSM49BAMEMB4xCzAJBgNVBAYT
        AlVTMQ8wDQYDVQQKDAZTUElGRkUwHhcNMTgwNTEzMTkzMzQ3WhcNMjMwNTEyMTkz
        MzQ3WjAeMQswCQYDVQQGEwJVUzEPMA0GA1UECgwGU1BJRkZFMHYwEAYHKoZIzj0C
        AQYFK4EEACIDYgAEWjB+nSGSxIYiznb84xu5WGDZj80nL7W1c3zf48Why0ma7Y7m
        CBKzfQkrgDguI4j0Z+0/tDH/r8gtOtLLrIpuMwWHoe4vbVBFte1vj6Xt6WeE8lXw
        cCvLs/mcmvPqVK9jo10wWzAdBgNVHQ4EFgQUh6XzV6LwNazA+GTEVOdu07o5yOgw
        DwYDVR0TAQH/BAUwAwEB/zAOBgNVHQ8BAf8EBAMCAQYwGQYDVR0RBBIwEIYOc3Bp
        ZmZlOi8vbG9jYWwwCgYIKoZIzj0EAwQDZwAwZAIwE4Me13qMC9i6Fkx0h26y09QZ
        IbuRqA9puLg9AeeAAyo5tBzRl1YL0KNEp02VKSYJAjBdeJvqjJ9wW55OGj1JQwDF
        D7kWeEB6oMlwPbI/5hEY3azJi16I0uN1JSYTSWGSqWc=
        -----END CERTIFICATE-----
         
="<unknown>" plugin_name=sconecas_sessionmanager plugin_type=SVIDStore subsystem_name=catalog
time="2021-11-12T12:30:31Z" level=error msg="cannot post SVID into CAS. Response status code =" 400 Bad Request="<unknown>" external=false plugin_name=sconecas_sessionmanager plugin_type=SVIDStore subsystem_name=catalog
time="2021-11-12T12:30:31Z" level=error msg="cannot post bundle into CAS " error={
  "msg": [
    "Session deserialization failed",
    "'spire-ca-spire-ca-<\\trust-bundle-session-name>' is not a valid session name\nCaused by: Only A-Z, a-z, 0-9, _, -, . and ~ are allowed at line 1 column 5"
  ]
}="<unknown>" external=false plugin_name=sconecas_sessionmanager plugin_type=SVIDStore subsystem_name=catalog
```